blocks/columns/columns.css

```diff
@@ -19,15 +20,122 @@
   display: block;
 }
 
-@media (min-width: 900px) {
+@media (min-width: 600px) {
   .columns > div {
-    align-items: center;
+    align-items: stretch;
     flex-direction: unset;
-    gap: 32px;
+    gap: 20px;
   }
 
   .columns > div > div {
     flex: 1;
     order: unset;
   }
 }
+
+@media (min-width: 900px) {
+  .columns > div {
+    gap: 100px;
+  }
+}
+
+/* thirds variant */
+
+.columns.thirds > div {
+  display: grid;
+  padding: 40px;
+}
+
+.columns.thirds > div > div:has(img) img {
```

Comment:
this selector I believe is functionally the same as the one on line 19: `.columns > div > .columns-img-col img ` so I'd suggest combining those into one rule.